apps/app/components/pages/create-update-block-inline.tsx

```diff
@@ -311,13 +312,14 @@ export const CreateUpdateBlockInline: React.FC<Props> = ({
 
                 return (
                   <TiptapEditor
+                    workspaceSlug={workspaceSlug as string}
                     ref={editorRef}
                     value={
                       value && value !== "" && Object.keys(value).length > 0
```

Comment:
Here, `value` and `watch("description_html")` hold the same value, double check is redundant. Can we remove `watch("description_html")`?

Reply:
Ohh yeah sure! I've done it!